recipes/strainscan/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "strainscan" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec4f51274a0b8b47e0c61d4dd9aec8decf7beea0f3037fcf90eeaab049e00142
+
+build:
+  number: 0
+  entry_points:
+    - strainscan = StrainScan.StrainScan:main
+    - strainscan_build = StrainScan.StrainScan_build:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python ==3.7.3
```

Comment:
It really needs this exact version of python?

Reply:
Yes. Cause the tool relies on one "cython" script, which requires the specific python version. However, as I tested, it can run normally with this exact version. In theory, 3.8>python>3.7 should be also fine, but I prefer to be conservative. Thus, I give the exact version to make sure this tool runs normally.

Reply:
If it needs cython then we can build it for anything >=3.7 if we add a C compiler. We'll get individual packages for each supported python version then.